src/components/Sidebar.tsx

```diff
@@ -49,7 +49,7 @@ const Sidebar = () => {
           ))}
         </div>
       </div>
-      <div className="w-full py-5 pl-5">
+      <div className="w-full px-5 py-5">
```

Comment:
Is there a reason we're not just using `p-5`?

Reply:
no, just overlooked it 🙂